common/templates/layouts/base.njk

```diff
@@ -119,6 +119,19 @@
   {% include "includes/messages.njk" %}
 {% endblock %}
 
+{% block footer %}
+    {{ govukFooter({
+      meta: {
+        items: [
+          {
+            href: FEEDBACK_URL,
+            text: t("feedback_link")
```

Comment:
We should keep the same logic where we only show this if the `FEEDBACK_URL` value is set.